[2025-04-29T17:43:23.486-0400] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: opportunity_pipeline.ingest_eventbrite scheduled__2025-04-28T21:00:00+00:00 [queued]>
[2025-04-29T17:43:23.490-0400] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: opportunity_pipeline.ingest_eventbrite scheduled__2025-04-28T21:00:00+00:00 [queued]>
[2025-04-29T17:43:23.490-0400] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-04-29T17:43:23.490-0400] {taskinstance.py:1280} INFO - Starting attempt 5 of 2
[2025-04-29T17:43:23.490-0400] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-04-29T17:43:23.495-0400] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): ingest_eventbrite> on 2025-04-28 21:00:00+00:00
[2025-04-29T17:43:23.498-0400] {standard_task_runner.py:55} INFO - Started process 35719 to run task
[2025-04-29T17:43:23.501-0400] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'opportunity_pipeline', 'ingest_eventbrite', 'scheduled__2025-04-28T21:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/opportunity_ingest_dag.py', '--cfg-path', '/var/folders/bc/zt35hlqn755d7y9zj64rxxph0000gn/T/tmpx_mnh3w4']
[2025-04-29T17:43:23.505-0400] {standard_task_runner.py:83} INFO - Job 19: Subtask ingest_eventbrite
[2025-04-29T17:43:23.543-0400] {task_command.py:388} INFO - Running <TaskInstance: opportunity_pipeline.ingest_eventbrite scheduled__2025-04-28T21:00:00+00:00 [running]> on host zihaos-macbook-pro.local
[2025-04-29T17:43:23.568-0400] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=you
AIRFLOW_CTX_DAG_ID=opportunity_pipeline
AIRFLOW_CTX_TASK_ID=ingest_eventbrite
AIRFLOW_CTX_EXECUTION_DATE=2025-04-28T21:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=5
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-04-28T21:00:00+00:00
[2025-04-29T17:45:14.989-0400] {local_task_job.py:112} ERROR - Received SIGTERM. Terminating subprocesses
[2025-04-29T17:45:15.005-0400] {process_utils.py:129} INFO - Sending 15 to group 35719. PIDs of all processes in the group: [35719]
[2025-04-29T17:45:15.006-0400] {process_utils.py:84} INFO - Sending the signal 15 to group 35719
[2025-04-29T17:45:19.030-0400] {taskinstance.py:1073} INFO - Dependencies not met for <TaskInstance: opportunity_pipeline.ingest_eventbrite scheduled__2025-04-28T21:00:00+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2025-04-29T17:45:19.032-0400] {taskinstance.py:1073} INFO - Dependencies not met for <TaskInstance: opportunity_pipeline.ingest_eventbrite scheduled__2025-04-28T21:00:00+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2025-04-29T17:45:19.033-0400] {local_task_job.py:147} INFO - Task is not able to be run
